python/lsst/pipe/tasks/dataFrameActions/__init__.py

```diff
@@ -0,0 +1,3 @@
+from ._actions import *
+from ._baseDataFrameActions import DataFrameAction
+from ._evalColumnExpression import *
```

Comment:
Are these star imports legit?

Reply:
Yeah, this is pretty normal for a `__init__` file, especially if the sub modules are "private"